src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentStatusFields.scala

```diff
@@ -0,0 +1,31 @@
+package uk.gov.nationalarchives.tdr.api.graphql.fields
+
+import io.circe.generic.auto._
+import sangria.marshalling.circe._
+import sangria.schema.{Argument, Field, IntType, OptionType, fields}
+import uk.gov.nationalarchives.tdr.api.auth.ValidateUserHasAccessToConsignment
+import uk.gov.nationalarchives.tdr.api.graphql.ConsignmentApiContext
+import uk.gov.nationalarchives.tdr.api.graphql.fields.FieldTypes.UuidType
+
+import java.time.ZonedDateTime
+import java.util.UUID
+
+object ConsignmentStatusFields {
+
+  val ConsignmentIdArg: Argument[UUID] = Argument("consignmentid", UuidType)
+  val mutationFields: List[Field[ConsignmentApiContext, Unit]] = fields[ConsignmentApiContext, Unit](
+    Field("setUploadConsignmentStatusValueToComplete", OptionType(IntType),
```

Comment:
😄  Yeah, I was going back and forth with this name because I wanted to make it as clear as possible, but it ended up being long.

1. "should I make the verb 'set' since other queries are 'get'"
2. and then "How about setUploadStatusToComplete or setUploadStatusToComplete" and some variations around this

But yes, your suggestion sounds good. How about `markUploadStatusAsCompleted` (I made it "completed" because I realised that the actual value in the db is "completed" not "complete").

Reply:
Yeah, I think the `get` names aren't really following the GraphQL naming conventions. They happened because we created them when we were still thinking in Java. 😅 

I have seen other examples of people naming queries things like `getFile`, but I think it's more common to just call that query `file`.